### PR TITLE
Update lisk node repository to use data from superchain registry

### PR DESCRIPTION
**Pull request by sameersubudhi**
### What was the problem?

This PR resolves #LISK-1114 and #LISK-1306

### How was it solved?

- [x] Remove unused rollup.json config for both mainnet and sepolia
- [x] Bump node dependencies
- [x] Update entrypoint scripts
- [x] Update default environment files
- [x] Update .gitignore
- [x] Update README

### How was it tested?

Locally against both testnet & mainnet with the following clients

- [x] op-geth 
- [x] op-reth

### CODE REVIEW

File: .env.sepolia [hunk 1]
```diff
@@ -1,4 +1,3 @@
-GENESIS_FILE_PATH=sepolia/genesis.json
```
Comment (matjazv) [inline .env.sepolia]: Isn't this still needed for Reth?
What about same variable inside `.env.mainnet`? It's still there.

File: geth/Dockerfile [hunk 2]
```diff
@@ -40,7 +40,5 @@ COPY supervisord.conf /etc/supervisor/conf.d/supervisord.conf
 COPY geth/geth-entrypoint ./execution-entrypoint
 COPY geth/download-apply-snapshot.sh .
 COPY op-node-entrypoint .
-COPY sepolia ./sepolia
-COPY mainnet ./mainnet
```
Comment (matjazv) [inline geth/Dockerfile:44]: Why is this removed?

File: reth/reth-entrypoint
Comment (matjazv) [inline reth/reth-entrypoint:1]: Experimental engine is default now, and the `--engine.experimental` flag is deprecated.

Comment (matjazv) [reply reth/reth-entrypoint:1]: See https://github.com/paradigmxyz/reth/releases/tag/v1.1.0

**Review by matjazv — APPROVED** [review]
Tested both Geth and Reth!
LGTM! Nice job! :rocket:

File: README.md [hunk 3]
```diff
@@ -126,14 +127,9 @@ Navigate to your `op-geth` directory and start service by running the command:
 
 ```sh
 ./build/bin/geth \
+    --op-network=$OP_NODE_NETWORK \
```
Comment (Incede) [inline README.md:116]: With this change i think the section above [`Initialize op-geth`](https://github.com/LiskHQ/lisk-node/pull/48/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R110-R122) becomes redundant and could be removed.

**Review by Incede — CHANGES_REQUESTED** [review]

**Review by Incede — APPROVED** [review]
